app/error/errorHandler.js

```diff
@@ -0,0 +1,18 @@
+export const errorHandler = (err) => {
+  const formattedError = {
+    status: err && err.status ? err.status : 400,
+    message: err && err.message ? err.message : 'Something went wrong',
+  };
+
+  if (err && err.response) {
+    formattedError.status = err.response.status;
+    formattedError.message = `${err.response.data.errors[0]} ${err.response.statusText}`;
+    if (err.response.status === 404) {
+      const splitUrl = err.response.config.url.split('/');
+      const solutionId = splitUrl[splitUrl.length - 2];
```

Comment:
should errorHandler know anything about Solution ids and its error message? If solution page url changes, that this have to be changed as well

Reply:
How generic we make the errorHandler is up for debate. I was just playing around with it. Due to the low complexity in this app I thought we could get away with this (as long as the api doesn't change it's responses). Alternatively we could ask them to return better error messages. Will discuss with everyone at later date.